runtime/cmd/cmd.go

```diff
@@ -99,9 +99,10 @@ func DefaultCheckerConfig(
 			importedLocation common.Location,
 			_ ast.Range,
 		) (sema.Import, error) {
-			if importedLocation == stdlib.CryptoChecker.Location {
+			cryptoChecker := stdlib.CryptoChecker()
+			if importedLocation == cryptoChecker.Location {
```

Comment:
Here and in other places, shouldn't `cryptoChecker := stdlib.CryptoChecker()` be moved inside the if condition?
Otherwise, the crypto contract would be loaded even if `importedLocation != cryptoChecker.Location`

Reply:
Great point! Refactored in 1d54a7106c0769a67459b7960f17b1485bdd8f58 by adding a constant for the location of the Crypto contract and using it for the check, then only creating the checker if necessary